apps/onboarding-example/src/App.js

```diff
@@ -0,0 +1,338 @@
+import React from "react"
+import "./App.css"
+import mondaySdk from "monday-sdk-js"
+import "monday-ui-react-core/dist/main.css"
+import Heading from "monday-ui-react-core/dist/Heading.js"
+import Chips from "monday-ui-react-core/dist/Chips.js"
+import TextField from "monday-ui-react-core/dist/TextField.js"
+import Button from "monday-ui-react-core/dist/Button.js"
+import Tooltip from "monday-ui-react-core/dist/Tooltip.js"
+import Icon from "monday-ui-react-core/dist/Icon.js"
+import LearnMore from "monday-ui-react-core/dist/icons/LearnMore"
+import Help from "monday-ui-react-core/dist/icons/Help"
+import IconButton from "monday-ui-react-core/dist/IconButton.js"
+
+const monday = mondaySdk();
+
+class App extends React.Component {
```

Comment:
I would really suggest to implement this as functional react component with hooks
Class component is so 2018 :)